db.json

```diff
@@ -4,22 +4,26 @@
       {
         "id": 1,
         "name": "architecture",
-        "description": "Some archy description."
+        "description": "Some archy description.",
+        "tooltip": "{proportion} of users have the {x} architecture"
```

Comment:
I'm also curious if the metric store can tap into the metric metadata info? I'll defer to @openjck on how best to structure that. Does it make sense to push some of this down to the metric API?